pkg/network/tracer/tracer_classification_test.go

```diff
@@ -54,6 +54,7 @@ type testContext struct {
 }
 
 func setupTracer(t *testing.T, cfg *config.Config) *Tracer {
+	cfg.BPFDebug = true
```

Comment:
Why force it here? one can control it by setting the env variable: `BPF_DEBUG=true`

Reply:
oops, removed